packages/core/auto-export.config.json

```diff
@@ -14,6 +14,11 @@
             "inputPath": "packages/*/src/index.ts",
             "outputPath": "packages/core/src/generated/checkout-button-strategies.ts",
             "memberPattern": "^create.+ButtonStrategy$"
+        },
+        {
+            "inputPath": "packages/*/src/index.ts",
+            "outputPath": "packages/core/src/generated/checkout-headless-button-strategies.ts",
+            "memberPattern": "^create.+HeadlessButtonStrategy$"
```

Comment:
Need to think about the right pattern for headless strategies. `test-core` **is failed** because for button strategies we have the same substring (`ButtonStrategy`) as for headless button strategies in `memberPattern` field. For this reason, we do not have a unique token. Should be replaced with one of the following examples: `HeadlessWalletStrategy`, `HeadlessStrategy` or something like that

Reply:
Maybe we can use `HeadlessWalletStrategy` here.